docs/stepbystep.md

```diff
@@ -50,7 +50,7 @@ interchanged between simulated devices and the Context Broker.
 
 The following list shows the prerequisite software and versions:
 
--   Orion Context Broker (v0.26)
+-   Orion Context Broker (v2.2.0)
 -   Node.js (v0.12.0)
```

Comment:
Node.js version should be also updated. I'd suggest v0.12.0 -> v8

Reply:
> Node.js version should be also updated. I'd suggest v0.12.0 -> v8



> Node.js version should be also updated. I'd suggest v0.12.0 -> v8

Ok.  I've updated the version of Node.js in stepbystep.md

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/410/commits/1242bba069de328326c1c789da27a43376d35292

(Please, whenever you address a comment, add a "Fixed in ..." mark as the above, for traceability reasons)

Reply:
> Fixed in [1242bba](https://github.com/telefonicaid/iotagent-json/commit/1242bba069de328326c1c789da27a43376d35292)
> 
> (Please, whenever you address a comment, add a "Fixed in ..." mark as the above, for traceability reasons)

Thanks for your guideline. I'll follow the same.